README.md

```diff
@@ -135,57 +146,40 @@ $ printf '\nexport PATH="%s:$PATH"\n' '${HOME}/.local/bin' | tee -a ~/.bashrc
 $ python3 -m pip install --user pipenv
 ```
 
-2. Clone the zulip/zulip-terminal repository locally
-```
-$ git clone git@github.com:zulip/zulip-terminal.git
-```
-
-3. Install dev requirements
+2. Install zulip-term, with the development requirements
 ```
-$ cd zulip-terminal
 $ pipenv --three
 $ pipenv install --dev
 $ pipenv run python setup.py develop
```

Comment:
Should we replace this with something like `pipenv run pip3 install -e .[dev]`?

Reply:
Assuming I understood the conclusion of the discussion, merging this as-is so that we have an improvement to the documentation present now, which we can incrementally improve upon.